Makefile

```diff
@@ -63,6 +63,14 @@ db-migrate:
 db-migrate-make:
 	@docker-compose exec --user=${USERNAMEDOCKER} back ./artisan make:migration ${ARGS}
 
+.PHONY: db-migrate-refresh
+db-migrate-refresh:
+	@docker-compose exec --user=${USERNAMEDOCKER} back ./artisan migrate:refresh
+
+.PHONY: db-migrate-refresh-seed
+db-migrate-refresh-seed:
+	@docker-compose exec --user=${USERNAMEDOCKER} back ./artisan migrate:refresh --seed
+
```

Comment:
O Fernando já tinha comentado sobre isso em outro PR que eu tinha levantado. Ele disse que não era uma boa ideia colocar comandos de rollback no Makefile, para evitar o risco de alguém executar esses comandos em produção.

Particularmente, não vejo tanto problema em ter esses comandos, pois se alguém for capaz de acessar a máquina de produção, ela conseguiria fazer qualquer coisa, tendo o Makefile ou não.

Agora eu acho que seria uma boa adição a esta sua PR é alterar o `migrate:rollback` para fazer o rollback de somente uma migration por execução, em vez de fazer o rollback de tudo de uma vez só. Em produção, pode ser uma dor de cabeça executar o rollback para querer desfazer uma tabela e remover todo o banco de dados.

Reply:
@bruno-s-campos, concordo tanto contigo, quanto com o @FernandoCostaB, no meu ponto de vista, em produção, só deveriam haver os comandos `up-prod` e um outro para executar as migrações e seeder, levando em consideração que em produção, não deveria ter acesso ao container justamente para evitar esse tipo de iteração e acabar causando problemas.